ocaml/sdk-gen/csharp/autogen/src/HTTP.cs

```diff
@@ -668,9 +668,9 @@ private static void AuthenticateProxy(ref Stream stream, Uri uri, IWebProxy prox
         }
 
 
-        private static Stream DoHttp(Uri uri, IWebProxy proxy, bool nodelay, int timeout_ms, params string[] headers)
+        private static Stream DoHttp(Uri uri, IWebProxy proxy, bool noDelay, int timeoutMs, params string[] headers)

```

Comment:
As discussed, let's use `WebHeaderCollection `

Reply:
I went for the dictionary in the end for the reason mentioned [here ](https://github.com/xapi-project/xen-api/pull/5302#discussion_r1446778061)